experiments/AMIP/modular/components/ocean/slab_seaice_init.jl

```diff
@@ -19,45 +19,63 @@ function slab_ice_space_init(::Type{FT}, space, p) where {FT}
     return Y
 end
 
+"""
+    ice_rhs!(du, u, p, _)
+
+Rhs method in the form as required by `ClimeTimeSteppers`, with the tendency vector `du`,
+the state vector `u` and the parameter vector, `p`, as input arguments.
 
-function ice_rhs!(du, u, p, t)
+This sea-ice energy formulation follows [Holloway and Manabe 1971](https://journals.ametsoc.org/view/journals/mwre/99/5/1520-0493_1971_099_0335_socbag_2_3_co_2.xml?tab_body=pdf),
+where sea-ice concentrations and thicknes are prescribed, and the model solves for temperature (curbed at the freezing point).
+"""
+function ice_rhs!(du, u, p, _)
     dY = du
     Y = u
     FT = eltype(dY)
 
     params = p.params
     F_aero = p.F_aero
     F_rad = p.F_rad
-    ice_mask = p.ice_mask
+    ice_fraction = p.ice_fraction
+    T_freeze = params.T_freeze
 
-    F_conductive = @. params.k_ice / (params.h) * (params.T_base - Y.T_sfc)
+    F_conductive = @. params.k_ice / (params.h) * (params.T_base - Y.T_sfc) # fluxes are defined to be positive when upward
     rhs = @. (-F_aero - F_rad + F_conductive) / (params.h * params.ρ * params.c)
-    parent(dY.T_sfc) .= parent(rhs) # apply_mask.(parent(ice_mask), >, parent(rhs), parent(rhs) .* FT(0), FT(0) )
+
+    # do not count tendencies that lead to temperatures above freezing, and mask out no-ice areas
+    unphysical = @. binary_mask.(T_freeze - (Y.T_sfc + FT(rhs) * p.dt), threshold = FT(0)) .*
+       binary_mask.(ice_fraction, threshold = eps())
+    parent(dY.T_sfc) .= parent(rhs .* unphysical)
 end
 
 """
-ice_init(::Type{FT}; tspan, dt, saveat, space, ice_mask, stepper = Euler()) where {FT}
+    ice_init(::Type{FT}; tspan, dt, saveat, space, ice_fraction, stepper = Euler()) where {FT}
 
 Initializes the `DiffEq` problem, and creates a Simulation-type object containing the necessary information for `step!` in the coupling loop.
 """
-function ice_init(::Type{FT}; tspan, saveat, dt, space, ice_mask, stepper = Euler()) where {FT}
+function ice_init(::Type{FT}; tspan, saveat, dt, space, ice_fraction, stepper = Euler()) where {FT}
 
     params = IceSlabParameters(
-        FT(2),
-        FT(1500.0),
-        FT(800.0),
-        FT(280.0),
-        FT(1e-3),
-        FT(1e-5),
-        FT(273.15),
-        FT(2.0),# k_ice
-        FT(0.8), # albedo
+        FT(2),  # ice thickness [m]
+        FT(900.0), # density of sea ice [kg / m3]
+        FT(2100.0), # specific heat of sea ice [J / kg / K]
+        FT(271.2), # temperature of sea water at the ice base
+        FT(1e-3), # roughness length for momentum [m]
+        FT(1e-5), # roughness length for tracers [m]
+        FT(271.2), # freezing point of sea water [K]
+        FT(2.0),# thermal condictivity of ice [W / m / K] (less in HM71)
+        FT(0.8), # sea ice albedo
```

Comment:
Maybe here you could just add comments of the field names in `IceSlabParameters`, after moving the more descriptive comments to the constructor itself.